addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/ImportDialog.java

```diff
@@ -176,6 +195,21 @@ private int getSelectedContextId() {
         return selectedContext.getId();
     }
 
+    private User getSelectedUser() {
+        var ext = getExtUserMgmt();
+        if (ext == null) {
+            return null;
+        }
```

Comment:
I know it's an unlikely situation but is it worth logging something here?

Reply:
The user will not be able to selecting any user if the extension is not enabled, we could log something earlier but if the user disabled the users extension I think it should not be a surprise that one will not be able to select users.